nifi-nar-bundles/nifi-extension-utils/nifi-record-utils/nifi-avro-record-utils/src/test/java/org/apache/nifi/avro/TestAvroTypeUtil.java

```diff
@@ -450,4 +450,22 @@ public void testBytesToStringConversion() {
         assertTrue(o instanceof String);
         assertEquals("Hello", o);
     }
+
+    @Test
+    public void testListToArrayConversion() {
+        final Charset charset = Charset.forName("UTF_32LE");
```

Comment:
`UTF-8` is what almost everyone uses so please use that.

Reply:
I copied the whole test setup from `testBytesToStringConversion`, not sure why that particular encoding was used there. Will change to UTF-8, this shouldn't affect anything anyway.